### ISSUE TITLE
Create a proof of concept reporting tool

### ISSUE THREAD

**Issue by sselph** [sselph/scraper#18]
Allow users to have the script parse their gamelist to upload missing hashes.

**Comment by robertybob** [comment]
Hi Sselph. I note that work is underway to incorporate your scraper into RetroPie by @joolswills and @herbfargus. I wondered whether if it would theoretically be possible / practical to do the same for the Reporter.exe you created (which isn't available to download on this GitHub?). 

For example; someone scrapes all their systems within RetroPie, finds quite a few blanks and wants to help improve your scraping tool by submitting the relevant missing hashes. Could these hashes be sent to just one editable Google Doc spreadsheet which someone with a bit of time on their hands like me could use to match up with TGDB IDs? I wouldn't mind helping as I figure that one day I may have a copy of that game currently missing from your database. 

If you'd prefer to keep your tool for just a few trusted people then that's understandable.

**Comment by sselph** [comment]
Thanks for the suggestion. I have been keeping it to trusted testers for now since I blindly add anything you send me. I assume you are certain the game named ABC is actually ABC and you've matched it to ABC in thegamesdb. For any ole person I'll want to probably factor in some trust mechanism so I know that the file names they provide are actually what the game is and just never figured it out.

**Comment by robertybob** [comment]
I've just had a better idea - what about only reporting those where the hash is found but no entry in TGDB can be found? At least that way it's going to be obvious whether the game is there or not.

**Comment by robertybob** [comment]
(Compared to whether someone's named the game correctly or not)

**Comment by sselph** [comment]
I actually have that complete list. It is at the bottom of the hash.csv file but is missing platform information. If you like I can output it to a sheet for you to take a look at.

**Comment by robertybob** [comment]
Yes please, I'll probably be useless when it comes to Japanese games mind you, but I'd certainly like to help :) Did you see my message about the PS1 games?
